airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -6668,7 +6668,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-jira:0.3.1"
+- dockerImage: "airbyte/source-jira:0.3.2"
```

Comment:
revert this change, it will be bumped automatically by publishing scripts

Reply:
Done :heavy_check_mark: